ios/Podfile.lock

```diff
@@ -638,6 +638,6 @@ SPEC CHECKSUMS:
   UMTaskManagerInterface: 2be431101b73604e64fbfffcf759336f9d8fccbb
   Yoga: 7ab6e3ee4ce47d7b789d1cb520163833e515f452
 
-PODFILE CHECKSUM: d2b0f4b53b059f1df00a0fbcc24f616cc29d453f
+PODFILE CHECKSUM: 45780cd8798839387522b3bc1e79a5ea1dc6499a
 
-COCOAPODS: 1.11.3
+COCOAPODS: 1.11.2
```

Comment:
Is this because of an M1 incompatibility with the later version that I read about or something else?